lib/modules/add-scoped-id.js

```diff
@@ -3,12 +3,27 @@ const postcss = require('postcss')
 const selectorParser = require('postcss-selector-parser')
 
 function isDeepCombinator (node) {
-  return node.type === 'combinator' && node.toString().trim() === '>>>'
+  if (node.type === 'combinator' && node.toString().trim() === '>>>') {
+    return true
+  }
+  if (node.type === 'tag' && node.toString().trim() === '/deep/') {
+    return true
+  }
+}
+
+function isSassDeepCombinator (node) {
```

Comment:
It would be better to choose other name because `/deep/` is not sass specific combinator.